app/models/package_manager/pypi/json_api_project.rb

```diff
@@ -137,7 +137,7 @@ def deprecation_status
         # on Releases. Make sure that the code here and in the VersionProcessor
         # do not conflict.
         # TODO: refactor this deprecation method to reuse the VersionProcessor status logic
-        if stable_releases.all?(&:yanked?)
+        if !stable_releases.empty? && stable_releases.all?(&:yanked?)
           is_deprecated = true
           message = latest_stable.yanked_reason
```

Comment:
Should this be a safe nav anyways?

Reply:
It should not be nil if `stable_releases` has data in it. I would rather not hide the error if this case does come up as that would mean there is a problem with mapping the upstream data to our release objects in here.